ocaml/typing/persistent_env.ml

```diff
@@ -167,16 +166,17 @@ let find_info_in_cache {persistent_structures; _} name =
 let find_in_cache penv name =
   find_info_in_cache penv name |> Option.map (fun ps -> ps.ps_val)
 
-let register_parameter ({param_imports; _} as penv) import =
+let register_parameter ({param_imports; _} as penv) modname =
+  let import = CU.Name.of_head_of_global_name modname in
```

Comment:
This ignores `modname.args` - do we statically know that should be empty?

Reply:
Added (with a note about how the _better_ thing would be to have `Global_module.Parameter_name.t`, which would avoid the runtime check)